Formula/tile38.rb

```diff
@@ -31,6 +31,11 @@ def post_install
     datadir.mkpath
   end
 
+  def caveats; <<~EOS
+    To connect: tile38-cli
```

Comment:
This does not seem specific to Homebrew, I recommend we simply drop the caveat.

Reply:
The PR is just about fixing component order.

Reply:
Sure, I am merely suggesting that we can improve it further and bring it in line with our policy :)

Reply:
but https://github.com/Homebrew/homebrew-core/blob/master/Formula/mysql.rb#L148-L149